src/applications/disability-benefits/526EZ/helpers.jsx

```diff
@@ -347,12 +348,14 @@ export const veteranInformationViewField = ({ formData }) => {
  */
 export const disabilityOption = ({ diagnosticCode, name, ratingPercentage }) => {
   // May need to throw an error to Sentry if any of these doesn't exist
+  // A valid rated disability *can* have a rating percentage of 0%
+  const showRatingPercentage = !!(ratingPercentage || ratingPercentage === 0);
```

Comment:
This works, but can be simplified with `Number.isInteger(ratingPercentage)`.

Reply:
Not supported in IE, though `babel-polyfill` probably takes care of this.

Reply:
Yeah, I think we can trust `babel-polyfill` to do that. The whole babel multiverse confuses me sometimes, but I did see a closed issue regarding this.